lib/msf/core/modules/external/shim.rb

```diff
@@ -144,6 +159,27 @@ def self.dos(mod)
     render_template('dos.erb', meta)
   end
 
+  def self.evasion(mod)
+    meta = mod_meta_common(mod)
```

Comment:
Based on the template, you will need to have the `payload_raw`, `payload_encoded`, and `target` options in the modules and drop them here with `meta = mod_meta_common(mod, ignore_options: ['payload_raw', 'payload_encoded', 'target'])`.